plugins/amdgpu/amdgpu_plugin_topology.c

```diff
@@ -58,6 +58,34 @@ bool kfd_vram_size_check;
 /* Preserve NUMA regions */
 bool kfd_numa_check;
 
+#define DRM_FIRST_RENDER_NODE 128
```

Comment:
how about we move all the #defines to a new header file.. there are a bunch of such definitions now

Reply:
definitions that are used in multiple files are inside the header. This one is only used here.